services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -6228,6 +6287,50 @@ func makeTransformersForDelete(numVersions uint64) []Transformer {
 	return res
 }
 
+func setupRepositoryTestWithDB(t *testing.T, dbConfig *DBConfig) (Repository, error) {
+	//t.Parallel()
```

Comment:
please add a comment, why this should not be run in parallel
```suggestion
	//t.Parallel()  // does not work in parallel because of...
```